modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/utils/utils.js

```diff
@@ -161,6 +161,17 @@ class utils {
 		}
 	}
 
+	/**
+	 * Logs the provided message in DEV environments
+	 * @param {String} message
+	 */
+	static log(message) {
+		if (process.env.NODE_ENV === 'development') {
+			// eslint-disable-next-line
+			console.log(message);
+		}
+	}
```

Comment:
One might argue that these logs are too chatty even for development. Maybe once we see this is stable we can remove them?

Tangential: why is this a `class` filled with `static` methods? It is never instantiated so it has no reason to be a class as far as I can tell.

Reply:
> One might argue that these logs are too chatty even for development. Maybe once we see this is stable we can remove them?

I could agree...

> Tangential: why is this a class filled with static methods? It is never instantiated so it has no reason to be a class as far as I can tell.

Classes were all the hype back then 😂

Reply:
![stay-classy](https://user-images.githubusercontent.com/7074/97974902-90817500-1dc8-11eb-9634-bac0ee59a6a6.gif)